cli/graph_util.rs

```diff
@@ -249,6 +249,18 @@ impl ModuleGraphCreator {
     package_configs: &[JsrPackageConfig],
     build_fast_check_graph: bool,
   ) -> Result<ModuleGraph, AnyError> {
+    fn graph_has_external_remote(graph: &ModuleGraph) -> bool {
+      // skip type checking when there are any external remote modules
+      // because those will be surfaced as errors later and would cause
+      // type checking to crash
+      graph.modules().any(|module| match module {
```

Comment:
This seems strange to me - do we just silently skip checking and there's no info surfaced about it? Or is it getting surfaced somewhere else?

Reply:
I'm going to update this comment. If the graph contains any of these then it's invalid.